src/models/primitive_wet.jl

```diff
@@ -94,7 +94,7 @@ $(TYPEDFIELDS)"""
     surface_heat_flux::SH = SurfaceHeatFlux(spectral_grid)
     surface_humidity_flux::HF = SurfaceHumidityFlux(spectral_grid)
     large_scale_condensation::LSC = ImplicitCondensation(spectral_grid)
-    convection::CV = SimplifiedBettsMiller(spectral_grid)
+    convection::CV = BettsMillerConvection(spectral_grid)
```

Comment:
that's another weird one that didn't get merged properly. I thought of git's merge a bit of black magic anyway and it seems there are indeed cases there it merges incorrectly or at least suggests something in a merge conflict that is actually a bit more complicated